packages/app-store/dailyvideo/lib/VideoApiAdapter.ts

```diff
@@ -56,10 +56,10 @@ const meetingTokenSchema = z.object({
 
 /** @deprecated use metadata on index file */
 export const FAKE_DAILY_CREDENTIAL: CredentialPayload & { invalid: boolean } = {
-  id: +new Date().getTime(),
+  id: 0,
   type: "daily_video",
   key: { apikey: process.env.DAILY_API_KEY },
-  userId: +new Date().getTime(),
+  userId: 0,
```

Comment:
Sometimes credentialId or userId should be saved to DB, but prisma will error if tries to save said values. I think we are better off using 0.
@zomars